dartagnan/src/main/java/com/dat3m/dartagnan/expression/integers/IntUnaryOp.java

```diff
@@ -0,0 +1,20 @@
+package com.dat3m.dartagnan.expression.integers;
+
+import com.dat3m.dartagnan.expression.ExpressionKind;
+
+public enum IntUnaryOp implements ExpressionKind {
+    CTLZ, MINUS;
+
+    @Override
+    public String toString() {
+        return getSymbol();
+    }
+
+    @Override
+    public String getSymbol() {
+        return switch (this) {
+            case CTLZ -> "ctlz ";
```

Comment:
do we really want the extra space here?

Reply:
yes, because this is printed as a unary operator `ctlz x`. Without the space it will print `ctlzx`.
We might want to come up with better printing though.